src/lib/cache.ts

```diff
@@ -28,8 +28,17 @@ export const defaultCacheFilename = '.ncu-cache.json'
 export const defaultCacheFile = `~/${defaultCacheFilename}`
 export const resolvedDefaultCacheFile = path.join(os.homedir(), defaultCacheFilename)
 
+/** Remove the default cache file. */
+export async function cacheClear() {
+  try {
+    fs.promises.rm(resolvedDefaultCacheFile)
```

Comment:
This should handle a custom `--cacheFile` also.

Reply:
It replicates `rm` behaviour in a cumbersome way. Perhaps a better design would be to store all the custom cache filepaths in the default cache and simply remove them altogether at `cacheClear`.

Reply:
Currently, the caching mechanism uses a single cache file, either the default or a specified file. It would make sense to me that `--cacheClear` would also operate on a single cache file, either the default or a specified file. So I would prefer that `--cacheFile` determine the cache file in both cases.

Reply:
Note that if you add `{ force: true }` you don't need to wrap it in a try-catch.